python/tests/compat/test_mantid.py

```diff
@@ -18,8 +18,9 @@ def mantid_is_available():
         return False
 
 
-@pytest.mark.skipif(not mantid_is_available(),
-                    reason='Mantid framework is unavailable')
+@pytest.mark.skip(reason='Mantid tests are skipped by default, as they '
```

Comment:
Why is this needed? Mantid is not available at the time the tests are run (you only add it to the docs environment, not the build environment).

Reply:
That's a very good point indeed!
Got my environments mixed up there, thanks!